coderd/database/dbauthz/methods_test.go

```diff
@@ -0,0 +1,367 @@
+package dbauthz_test
```

Comment:
Thoughts on calling this `helper_test.go` instead? Seems more like helpers and setup rather than `methods`. Or `setup_test.go` would work too!